docs/api/ipc-main-process.md

```diff
@@ -45,4 +46,4 @@ Assign to this to return an value to synchronous messages.
 
 ### Event.sender
 
-The `WebContents` of the web page that has sent the message.
+The `WebContents` of the renderer that has sent the message.
```

Comment:
It is possible for a renderer to have multiple `WebContents`, we should just remove "of the renderer".